Library/include/CSP/Systems/Quota/QuotaSystem.h

```diff
@@ -83,12 +83,12 @@ class CSP_API CSP_NO_DISPOSE QuotaSystem : public SystemBase
 	/// @param TierName TierNames : Name of the tier
 	/// @param FeatureName TierFeatures : Name of the feature
 	/// @param Callback FeatureQuotaCallback : callback when asynchronous task finishes
-	CSP_ASYNC_RESULT void GetTierFeatureQuota(const TierNames& TierName, const TierFeatures& FeatureName, FeatureQuotaCallback Callback);
+	CSP_ASYNC_RESULT void GetTierFeatureQuota(const TierNames TierName, const TierFeatures FeatureName, FeatureQuotaCallback Callback);
```

Comment:
`const` is not needed here due to this being passed by value.

Reply:
Yup agreed, thanks!

Reply:
Fixed in latest.